.watchmanconfig

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Note that an empty `.watchmanconfig` on an FB-laptop will actually not work, and crashes everything. I'm making this update in this diff so that FB engineers can develop Jest in peace :)